src/regolith/schemas.json

```diff
@@ -2962,37 +2962,48 @@
             "type": "list",
             "schema": {
                 "major": {
-                    "description": "The major update of the software at the release.",
+                    "description": "The major release number of the software at the release.",
                     "required": true,
                     "type": "int"
                 },
                 "minor": {
-                    "description": "The minor update of the software at the release.",
+                    "description": "The minor release number of the software at the release.",
                     "required": true,
                     "type": "int"
                 },
                 "patch": {
-                    "description": "The patch update of the software at the release.",
+                    "description": "The patch number of the software at the release.",
                     "required": true,
                     "type": "int"
                 },
+                "release_type": {
+                    "description": "The type of release: 'major', 'minor', 'patch', or 'pre-release'.",
+                    "required": true,
+                    "type": "string",
+                    "enum": ["major", "minor", "patch", "pre-release"]
```

Comment:
please check elsewhere in schema, I think we are using a different syntax for this than enum.  We may want to switch to this as it is more standard json schema, but we are using a different schema validation system.  Please look elsewhere in the `schema.json` for how we are doing this.